.eslintrc.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     '.yarn/**',
     '**/build/*',
     '**/coverage/*',
-    '**/node_modules/*'
+    '**/node_modules/*',
+    '**/config.ts'
```

Comment:
just checking which file were you planning to exclude?
`packages/extension-chains/src/config.ts` still included

Reply:
excluded it... had few listing err in it... mostly related to sort-key in types..
but in case of enum we don't really want to sort...hv added at the top of the file /* eslint sort-keys: 0 */